docs/source/user_guide/01a_interface.rst

```diff
@@ -0,0 +1,29 @@
+
+nbgrader interface highlights
+=============================
+
+Instructor toolbar extension for Jupyter notebooks
+--------------------------------------------------
+The nbgrader toolbar extension for Jupyter notebooks guides the instructor through
+assignment and grading tasks using the familiar Jupyter notebook interface.
+
+![Creating assignment](docs/source/user_guide/images/creating_assignment.gif "Creating assignment")
```

Comment:
Looks like this is one of the files that's still in markdown rather than rst. You should be able to convert it pretty easily with pandoc, e.g. `pandoc --from markdown --to rst 01a_interface.rst`